src/main/java/com/github/instagram4j/instagram4j/requests/media/ConfigureToIgtvRequest.java

```diff
@@ -0,0 +1,56 @@
+package com.github.instagram4j.instagram4j.requests.media;
+
+import com.github.instagram4j.instagram4j.IGClient;
+import com.github.instagram4j.instagram4j.models.IGPayload;
+import com.github.instagram4j.instagram4j.requests.IGPostRequest;
+import com.github.instagram4j.instagram4j.responses.IGResponse;
+import com.github.instagram4j.instagram4j.utils.IGUtils;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import okhttp3.Request;
+
+@RequiredArgsConstructor
+@AllArgsConstructor
+public class ConfigureToIgtvRequest extends IGPostRequest<IGResponse> {
```

Comment:
Change Class name to `MediaConfigureToIgtvRequest` to match the `path()` endpoint name

Reply:
Good catch! Furthermore, it looks like there was already a `MediaConfigureToIgtvRequest` that I have adapted instead. 👍